ui/src/components/layer/options/layer-options.ts

```diff
@@ -68,45 +72,71 @@ export class NgmLayerOptions extends LitElementI18n {
   }
 
   readonly render = () => html`
-    <div class="ngm-base-layer">
-      <div
-        title=${!this.hideExaggeration ? i18next.t('dtd_hide_exaggeration') : i18next.t('dtd_show_exaggeration')}
-        class="ngm-layer-icon ${classMap({
-          'ngm-visible-icon': !this.hideExaggeration,
-          'ngm-invisible-icon': this.hideExaggeration,
-        })}"
-        @click=${() => {
-          if (!this.viewer) return;
-          this.hideExaggeration = !this.hideExaggeration;
-          const exaggeration = this.hideExaggeration ? 1 : this.exaggeration;
-          this.viewer.scene.verticalExaggeration = exaggeration;
-          this.updateExaggerationForKmls();
-          NavToolsStore.exaggerationChanged.next(exaggeration);
-          this.viewer.scene.requestRender();
-        }}>HIDE
+      <div class="group">
+        <ngm-core-icon
+          icon="${this.hideExaggeration ? 'invisible' : 'visible'}"
+          title=${!this.hideExaggeration ? i18next.t('dtd_hide_exaggeration') : i18next.t('dtd_show_exaggeration')}

```

Comment:
Invert this ternary.
```suggestion
          title=${this.hideExaggeration ? i18next.t('dtd_show_exaggeration') : i18next.t('dtd_hide_exaggeration')}

```